### PR TITLE
[ticket/14924] changes core.user_add_modify_data

### PR DESCRIPTION
**Pull request by david63**
Checklist:

- [ ] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [ ] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-12345

### CODE REVIEW

File: phpBB/includes/functions_user.php [hunk 1]
```diff
@@ -272,13 +272,19 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
...
 	* @event core.user_add_modify_data
-	* @var array	user_row		Array of user details submited to user_add
-	* @var array	cp_data			Array of Custom profile fields submited to user_add
-	* @var array	sql_ary		Array of data to be inserted when a user is added
+	* @var array	user_row			Array of user details submited to user_add
+	* @var array	cp_data				Array of Custom profile fields submited to user_add
+	* @var array	sql_ary				Array of data to be inserted when a user is added
 	* @since 3.1.0-a1
 	* @change 3.1.0-b5
+	* @var array	notifications_data	Array of notification data to be inserted when a user is added
```
Comment (VSEphpbb) [inline phpBB/includes/functions_user.php]: This is in the wrong position

Comment (david63) [inline phpBB/includes/functions_user.php]: Where should it be?

Comment (marc1706) [inline phpBB/includes/functions_user.php]: Please add it right below line 277.

File: phpBB/includes/functions_user.php [hunk 2]
```diff
@@ -272,13 +272,19 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
...
+	* @change 3.2.0
```
Comment (marc1706) [inline phpBB/includes/functions_user.php]: Use `@changed` and state what you changed. Target version should be 3.2.0-RC3.

**Comment by marc1706** [comment]
Slightly confused by your changes now. Did you mean to submit this for 3.1.11-RC1 or 3.2.0-RC3?

**Comment by david63** [comment]
Sorry Mark - I confuse myself at times!

I definitely want it in 3.2.0-RC£ but I don't mind if it goes in 3.1.11-RC1 as well

File: phpBB/includes/functions_user.php [hunk 3]
```diff
@@ -275,10 +275,10 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
 	* @var array	user_row			Array of user details submited to user_add
 	* @var array	cp_data				Array of Custom profile fields submited to user_add
 	* @var array	sql_ary				Array of data to be inserted when a user is added
+	* @var array	notifications_data	Array of notification data to be inserted when a user is added
 	* @since 3.1.0-a1
 	* @change 3.1.0-b5
-	* @var array	notifications_data	Array of notification data to be inserted when a user is added
-	* @change 3.2.0
+	* @change 3.1.11-a1
```
Comment (3D-I) [inline phpBB/includes/functions_user.php]: `* @changed 3.1.11-RC1`

**Comment by VSEphpbb** [comment]
Your latest commit message was wrong. It needs to be amended to follow the proper format.

File: phpBB/includes/functions_user.php [hunk 4]
```diff
@@ -272,13 +272,19 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
...
-	* @var array	user_row		Array of user details submited to user_add
-	* @var array	cp_data			Array of Custom profile fields submited to user_add
-	* @var array	sql_ary		Array of data to be inserted when a user is added
+	* @var array	user_row			Array of user details submited to user_add
+	* @var array	cp_data				Array of Custom profile fields submited to user_add
+	* @var array	sql_ary				Array of data to be inserted when a user is added
+	* @var array	notifications_data	Array of notification data to be inserted when a user is added
 	* @since 3.1.0-a1
 	* @change 3.1.0-b5
+	* @change 3.1.11-a1
```
Comment (marc1706) [inline phpBB/includes/functions_user.php]: Needs to be `@changed` and you also should add what was changed in the event, i.e. `Added notifications data array`

Comment (marc1706) [inline phpBB/includes/functions_user.php]: Also, the current target version is 3.1.11-RC1

**Comment by marc1706** [comment]
Closing in favor of #4706

**Comment by marc1706** [comment]
Thanks for your contribution. I've taken the liberty to fix the remaining issues and created a separate PR for that.

**Pull request closed without merging**